ftl/BUILD

```diff
@@ -100,13 +91,22 @@ py_binary(
 py_image(
     name = "node_builder_image",
     srcs = [":node_lib"],
-    base = ":node_builder_base",
+    base = "@debian_docker//reproducible/ubuntu:nodejs_ubuntu_build",
     main = "node/main.py",
     deps = [
         "@containerregistry",
     ],
 )
 
+# The node base image sets a default CMD, which causes issues if you run our image with no CMD.
```

Comment:
I think you can actually just delete this rule since you're not using it anymore.

Reply:
its used for testing

Reply:
https://github.com/aaron-prindle/runtimes-common/blob/ca9a0946b9890967ac4b0d2ef5b343c9f0dc1f48/ftl/BUILD#L113